depends/packages/openssl.mk

```diff
@@ -0,0 +1,45 @@
+# Pretty much taken verbatim from Bitcoin Core
```

Comment:
Bitcoin chose not to take this update. Several distros have patched around it as well. See https://github.com/bitcoin/bitcoin/pull/5929